snapshots/output/multi-project/src/example.ts

```diff
@@ -0,0 +1,9 @@
+  export function hello() {
+//                ^^^^^ definition example 1.0.0 src/example.ts/hello().
+    const a = 'a'
+//        ^ definition local 2
```

Comment:
Why is this `local 2` instead of `local 0`? It looks like a new `Visitor` is created per file, so the counter should restart from 0, right?